test/e2e/installplan_e2e_test.go

```diff
@@ -2715,6 +2715,273 @@ var _ = Describe("Install Plan", func() {
 			Expect(ref.Kind).To(Equal("ConfigMap"))
 		}
 	})
+	It("uses the OLM client for CRD installation", func() {
+		c := newKubeClient()
+		crc := newCRClient()
+
+		By("using a scoped serviceaccount specifified in the operatorgroup")
+		ns, err := c.KubernetesInterface().CoreV1().Namespaces().Create(context.TODO(), &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: genName("ns-"),
+			},
+		}, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		defer c.KubernetesInterface().CoreV1().Namespaces().Delete(context.TODO(), ns.GetName(), metav1.DeleteOptions{})
+
+		// create SA
+		sa := &corev1.ServiceAccount{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      genName("sa-"),
+				Namespace: ns.GetName(),
+			},
+		}
+		_, err = c.KubernetesInterface().CoreV1().ServiceAccounts(ns.GetName()).Create(context.TODO(), sa, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		// create role that includes permission for installing the kiali operator, but not CRDs
+		// see https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/scoped-operator-install.md
+		role := &rbacv1.ClusterRole{
```

Comment:
I decided to go with this kiali bundle because I didn't want to introduce a new image dependency for this test. It's also representative of an actual operator installation so I think it's potentially helpful in future tests.

I agree that the RBAC is rather bloated, but I'd rather not rewrite the test solely for this reason. Since this is test code it's not in the compiled binary anyway.  

I could move the RBAC to another file if that's helpful

Reply:
> write a CRD to an InstallPlan's status directly (forgo resolution)

I'd like to see this, too. If the behavior verified by this test is CRD creation in the presence of a scoped SA, then doing resolution and unpacking makes the test itself more fragile (i.e., it can fail in many ways that do not indicate a problem with the behavior under test).

Reply:
Alright, I will rewrite the e2e test to make an installplan instead